spec.html

```diff
@@ -2805,32 +2805,64 @@
       </emu-note>
       <p>A <dfn>Reference</dfn> is a resolved name or property binding. A Reference consists of three components, the base value component, the referenced name component, and the Boolean-valued strict reference flag. The base value component is either *undefined*, an Object, a Boolean, a String, a Symbol, a Number, or an Environment Record. A base value component of *undefined* indicates that the Reference could not be resolved to a binding. The referenced name component is a String or Symbol value.</p>
       <p>A <dfn id="super-reference">Super Reference</dfn> is a Reference that is used to represents a name binding that was expressed using the super keyword. A Super Reference has an additional thisValue component, and its base value component will never be an Environment Record.</p>
-      <p>The following abstract operations are used in this specification to access the components of references:</p>
-      <ul>
-        <li>
-          <dfn aoid="GetBase" id="ao-getbase">GetBase</dfn>(_V_). Returns the base value component of the reference _V_.
-        </li>
-        <li>
-          <dfn aoid="GetReferencedName" id="ao-getreferencedname">GetReferencedName</dfn>(_V_). Returns the referenced name component of the reference _V_.
-        </li>
-        <li>
-          <dfn aoid="IsStrictReference" id="ao-isstrictreference">IsStrictReference</dfn>(_V_). Returns the strict reference flag of the reference _V_.
-        </li>
-        <li>
-          <dfn aoid="HasPrimitiveBase" id="ao-hasprimitivebase">HasPrimitiveBase</dfn>(_V_). Returns *true* if Type(_V_'s base value component) is Boolean, String, Symbol, or Number; otherwise returns *false*.
-        </li>
-        <li>
-          <dfn aoid="IsPropertyReference" id="ao-ispropertyreference">IsPropertyReference</dfn>(_V_). Returns *true* if either the base value component of the reference _V_ is an object or HasPrimitiveBase(_V_) is *true*; otherwise returns *false*.
-        </li>
-        <li>
-          <dfn aoid="IsUnresolvableReference" id="ao-isunresolvablereference">IsUnresolvableReference</dfn>(_V_). Returns *true* if the base value component of the reference _V_ is *undefined*; otherwise returns *false*.
-        </li>
-        <li>
-          <dfn aoid="IsSuperReference" id="ao-issuperreference">IsSuperReference</dfn>(_V_). Returns *true* if the reference _V_ has a thisValue component; otherwise returns *false*.
-        </li>
-      </ul>
       <p>The following abstract operations are used in this specification to operate on references:</p>
 
+      <emu-clause id="sec-getbase" aoid="GetBase">
```

Comment:
I'm going to leave this commit behind for now. I need to preserve these IDs most likely. Could give the clauses IDs like the dfn IDs (ao-isstrictreference, etc.) or add contentless anchors like a few other places. Any preference?